x-pack/plugins/cloud_security_posture/public/components/fleet_extensions/csp_boxed_radio_group.tsx

```diff
@@ -58,6 +59,8 @@ export const RadioGroup = ({
             anchorProps={{
               style: {
                 flexGrow: 1,
+                /* Set Maximum width ONLY for Integration type buttons, we know this because only Integration Type has icons */
+                ...(option.icon && { maxWidth: `calc(${euiTheme.size.xxs} * 115)` }),
```

Comment:
we can also use the flex CSS property to achieve that:

```js
        style: {
          flex: '1 1 0',
        },
```

for context, `flex: '1 1 0'` is a shorthand representation of these three values in the order of flex-grow, flex-shrink, and flex-basis. 

So, `flex: '1 1 0'` means that the flex item will grow and shrink proportionally with other items with an initial size of zero, making all of the elements achieve the same width.